src/features/events_list/atoms/eventListResource.ts

```diff
@@ -1,18 +1,37 @@
-import { createResourceAtom } from '~utils/atoms';
+import { createResourceAtom, combineAtoms } from '~utils/atoms';
 import { apiClient } from '~core/apiClientInstance';
 import { autoRefreshService } from '~core/autoRefreshServiceInstance';
 import { currentEventFeedAtom } from '~core/shared_state';
 import type { Event } from '~core/types';
+import { eventListFilters } from './eventListFilters';
+
+const depsAtom = combineAtoms({
+  currentFeed: currentEventFeedAtom,
+  filters: eventListFilters,
+});
 
 export const eventListResourceAtom = createResourceAtom(
-  async (currentFeed) => {
-    const params =
-      currentFeed && currentFeed.id ? { feed: currentFeed.id } : undefined;
-    const responseData = await apiClient.get<Event[]>(`/events/`, params, true);
+  async (depsAtomState) => {
+    const deps = depsAtomState!;
```

Comment:
we can do 
```

    const deps = depsAtomState;

    const params: {
      feed?: string;
      bbox?: string;
    } = {
      feed: deps?.currentFeed?.id,
      bbox: deps?.filters.bbox?.join(','),
    };
```

Reply:
> doesn't eslint alerts you here? 
It is. Right now i'am sure that deps atom state can't be null, but typescript can't see that. I decide to avoid extra runtime checking in that case

Reply:
ok, got it 👌

Reply:
Hm, probably i wrong, will use checks